### PR TITLE
make editor accessable globaly for customizing

### PR DESCRIPTION
**Pull request by KALA-Typo3**
For adding custom blocks and changing behavior with a custom plugin

### CODE REVIEW

**Comment by RCheesley** [comment]
@cla-bot check

Thanks for the PR @KALA-Typo3 - Assume this is a parallel PR for https://github.com/mautic/mautic/pull/10510?

File: Assets/library/js/builder.service.js
Comment (RCheesley) [inline Assets/library/js/builder.service.js]:
```suggestion
    // make editor accessible globally for customizing
``` 

I know it's only a comment, but a couple of typos to fix ;)

**Review by RCheesley — CHANGES_REQUESTED** [review]
Thanks for the PRs - just a small typo suggestion on the comment. Will defer to @adiux for code review before we merge.

**Comment by KALA-Typo3** [comment]
Yes, I indeed remember posting it on a 3.x branch before. I also signed the agreement, since I want to contribute more often.

Sorry for typos, didn't double check before comitting.

**Comment by adiux** [comment]
@RCheesley I guess we can close this in favor of https://github.com/mautic/mautic/pull/10781

**Pull request closed without merging**
